src/graphql/resolvers/resolverHelper.js

```diff
@@ -17,4 +17,30 @@ const GetMany = async (db, table) => {
   return results.Items || [];
 };
 
-export { GetSingle, GetMany };
+const GetFiltered = async (db, table, filters) => {
```

Comment:
Would it make more sense to call this `dynamoScan()` or something like that?

Reply:
In that vein, these "helpers" really encapsulate dynamoDB methods. Perhaps we rename them to `dynamoQuery` `dynamoScan` or `dynamoGet`? Or if we want to hide the implementation just `dbQuery` `dbGet` and `dbScan`?

Just a suggestion